tcii18n/translator.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+
+class Translator:
+    def __init__(self, filename):
+        self.reader = csv.reader(open(filename, 'r'))
```

Comment:
If I close the file when I read it, I have to reopen it everytime, like the first solution, even in the example, files are opened everytime